tests/conftest.py

```diff
@@ -11,12 +11,14 @@
 from pytest import fixture
 from sha3 import sha3_256
 
-from bigchaindb_driver.common.transaction import Transaction, _fulfillment_to_details
-
+from bigchaindb_driver.common.transaction import _fulfillment_to_details
+from bigchaindb_driver.common.transaction import Transaction
```

Comment:
Why did you split the imports here?

Reply:
it raised a flake8 warning, so i updated it with a parenthesis approach for multiple imports